vault/resource_kubernetes_auth_backend_config.go

```diff
@@ -61,6 +61,12 @@ func kubernetesAuthBackendConfigResource() *schema.Resource {
 					return strings.Trim(v.(string), "/")
 				},
 			},
+			"issuer": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "kubernetes/serviceaccount",
```

Comment:
Since this default is set on _Vault's_ side, would it be possible to strip the default here (or set it to "") and leave the defaulting up to Vault? That way if it ever changes, there's no need to sync it here.

Reply:
PTAL: https://github.com/terraform-providers/terraform-provider-vault/pull/601/commits/ab0e44add8b1eb775c109207a068a1ac6c0fb317